R/pkg/R/functions.R

```diff
@@ -67,6 +67,14 @@ createFunctions <- function() {
 
 createFunctions()
 
+#' @rdname functions
+#' @return Creates a Column class of literal value.
+#' @export
+lit <- function(x) {
```

Comment:
I didn't know we can use `ANY` as a signature.

"There are two special classes that can be used in the signature: missing and ANY. missing matches the case where the argument is not supplied, and ANY is used for setting up default methods."
http://adv-r.had.co.nz/S4.html